--- gcc-3.4.6/gcc/version.c	2006-03-06 04:49:27.000000000 +0100
+++ gcc-3.4.6-patched/gcc/version.c	2017-10-18 22:09:49.506277500 +0200
@@ -5,7 +5,7 @@
    please modify this string to indicate that, e.g. by putting your
    organization's name in parentheses at the end of the string.  */
 
-const char version_string[] = "3.4.6";
+const char version_string[] = "3.4.6b";
 
 /* This is the location of the online document giving instructions for
    reporting bugs.  If you distribute a modified version of GCC,
